src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const cleanSource = sourceString.trim();
+
+  const declarations = cleanSource.split(';');
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have same indentation. This makes code much more readable. Although not chained here, consider applying a similar principle to split the string and filter out empty declarations in a more readable format.